### DISCUSSION TITLE
Is it possible to nest x-show inside x-if?

### DISCUSSION THREAD

**Discussion by lasseeee** [alpinejs/alpine#1208] [1. Help]
I find it impossible to get a nested `x-show` inside an `x-if` to work, or any other element inside the `x-if` for that matter (datepickers etc.), after displaying the DOM through `x-if`.

Should this be possible, or am I hitting some obvious limit with my implementation?

**Comment by KevinBatdorf** [comment]
Can you share a demo of it broken?
